scripts/minikube.sh

```diff
@@ -216,16 +213,16 @@ up)
     if minikube_supports_psp; then
         enable_psp
         # shellcheck disable=SC2086
-        ${minikube} start --force --memory="${MEMORY}" --cpus="${CPUS}" -b kubeadm --kubernetes-version="${KUBE_VERSION}" --driver="${VM_DRIVER}" --feature-gates="${K8S_FEATURE_GATES}" --cni="${CNI}" ${EXTRA_CONFIG}
+        ${minikube} start --force --memory="${MEMORY}" --cpus="${CPUS}" -b kubeadm --kubernetes-version="${KUBE_VERSION}" --driver="${VM_DRIVER}" --cni="${CNI}" ${EXTRA_CONFIG}
```

Comment:
what if someone wants to pass a newly added feature gate?

Reply:
Got it :+1: ,Made some changes to preserve capability to set feature gates option with default `"ExpandCSIVolumes=true"` (still in beta).